Documentation/operations/upgrade.rst

```diff
@@ -320,6 +320,12 @@ Helm Options
   ``podSecurityContext``.
 * The ``securityContext`` for Hubble Relay now defaults to drop all
   capabilities and run as non-root user.
+* The ``hubble.peerService.enabled`` value is deprecated. Currently ``true`` by
+  default, Cilium v1.15 will remove the option to set it to ``false`` as support
```

Comment:
```suggestion
  default, Cilium v1.14 will remove the option to set it to ``false`` as support
```

Reply:
Removing this as it won't affect the v1.14 release notes, will add it back to the v1.13 backport PR.